en-US/firefox/adblocker.lang

```diff
@@ -0,0 +1,131 @@
+## NOTE: https://www-dev.allizom.org/firefox/features/adblocker
```

Comment:
Add another line:
`## NOTE: https://www-dev.allizom.org/%LOCALE%/firefox/features/adblocker`

This will allow you to see the localized page by locale on staging: https://l10n.mozilla-community.org/langchecker/?locale=all&website=0&file=firefox/best-browser.lang. Example of what you did before https://github.com/mozilla-l10n/www.mozilla.org/blob/master/en-US/firefox/best-browser.lang.

Reply:
Don't forget to add this 2nd Note.

Reply:
Done & done :)